.github/workflows/update-cache.yml

```diff
@@ -8,7 +8,7 @@ concurrency:
 on:
   workflow_dispatch:
   schedule:
-    - cron: 0 */12 * * *
+    - cron: 0 */3 * * *
```

Comment:
the cache update won't do anything when there's no update so it's reasonable imo to run this more frequently so the update happens faster in case there is an update.